src/data/tattoos.txt

```diff
@@ -0,0 +1,306 @@
+1
```

Comment:
New file, versioned according to KoLmafia idiom, and tab delimited?  Can't really verify the latter.